kafka/channel/pkg/reconciler/dispatcher/kafkachannel.go

```diff
@@ -98,6 +98,8 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 		KnCEConnectionArgs: connectionArgs,
 		ClientID:           "kafka-ch-dispatcher",
 		Brokers:            kafkaConfig.Brokers,
+		TlsName:            kafkaConfig.TlsName,
+		TlsNamespace:       kafkaConfig.TlsNamespace,
```

Comment:
I think as others suggested, we should change the name.
`TlsSecretName/TlsSecretNamespace`?

Reply:
Oh, just saw `AuthSecretName` suggestion. That sounds better